lib/middleware/session.js

```diff
@@ -79,7 +79,14 @@ var ServletSession = exports.ServletSession = function(request) {
                 // to the attributes in the servlet session object.
                 data = new JavaAdapter(org.mozilla.javascript.NativeObject, {
                     put: function(name, start, value) {
-                        getSession().setAttribute(name, value);
+                    	if (typeof(value) === 'undefined' || value === null) {
+                    		getSession().removeAttribute(name);
+                    		if (Object.prototype[name]) {
+                                Object.prototype[name] = value;
```

Comment:
@ehoogerbeets since I think we should avoid any modification of `Object.prototype`: can we drop these lines or do you see a use for them?